src/store/modules/order/mutation-types.ts

```diff
@@ -0,0 +1,2 @@
+export const SN_ORDER = 'order'
+export const ORDER_COMPLETED_UPDATED = SN_ORDER + '/COMPLETED_CHANGED'
```

Comment:
```suggestion
export const ORDER_COMPLETE_UPDATED = SN_ORDER + '/COMPLETE_UPDATED'
```

Reply:
Thanks, @ymaheshwari1 Sir for your suggestion I have changed the variable name and it's property name